[2016-09-26 01:11:23] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 01:11:51] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 01:12:14] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 01:12:21] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 01:34:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 08:54:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 08:54:21] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /hello"" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /hello\" at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/api/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 21:59:16] request.INFO: Matched route "homepage". {"route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://localhost/ivq-rest/web/"} []
[2016-09-26 21:59:16] security.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2016-09-26 21:59:16] request.CRITICAL: Uncaught PHP Exception Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: "You have requested a non-existent service "fos_user.user_manager"." at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php line 305 {"exception":"[object] (Symfony\\Component\\DependencyInjection\\Exception\\ServiceNotFoundException(code: 0): You have requested a non-existent service \"fos_user.user_manager\". at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Container.php:305)"} []
[2016-09-26 21:59:16] security.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2016-09-26 22:10:34] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /api/doc"" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /api/doc\" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 22:10:45] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /api/doc"" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /api/doc\" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:39)"} []
[2016-09-26 22:12:06] request.INFO: Matched route "index". {"route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_format":"json","_route":"index"},"request_uri":"http://localhost/ivq-rest/web/login?_format=json&password=tmp&username=tmp"} []
[2016-09-26 22:12:06] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Security\Core\Exception\AccessDeniedException: "No matching user account found" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/src/AppBundle/Controller/DefaultController.php line 54 {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AccessDeniedException(code: 403): No matching user account found at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/src/AppBundle/Controller/DefaultController.php:54)"} []
[2016-10-06 11:48:13] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/ivq-rest/")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/ivq-rest/\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:93)"} []
[2016-10-06 11:51:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/ivq-rest/")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/ivq-rest/\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:93)"} []
[2016-10-13 15:10:07] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /" (from "http://localhost/ivq-rest/")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" (from \"http://localhost/ivq-rest/\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:93)"} []
[2016-10-14 18:16:25] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Persistence\Mapping\MappingException: "Class 'Acme\ApiBundle\Entity\User' does not exist" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96 {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:16:25] request.CRITICAL: Exception thrown when handling an exception (Doctrine\Common\Persistence\Mapping\MappingException: Class 'Acme\ApiBundle\Entity\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96) {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:19:03] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Persistence\Mapping\MappingException: "Class 'Acme\ApiBundle\Entity\User' does not exist" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96 {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:19:03] request.CRITICAL: Exception thrown when handling an exception (Doctrine\Common\Persistence\Mapping\MappingException: Class 'Acme\ApiBundle\Entity\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96) {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:26:00] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Persistence\Mapping\MappingException: "Class 'Acme\ApiBundle\Entity\User' does not exist" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96 {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:26:00] request.CRITICAL: Exception thrown when handling an exception (Doctrine\Common\Persistence\Mapping\MappingException: Class 'Acme\ApiBundle\Entity\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96) {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:29:25] request.CRITICAL: Uncaught PHP Exception Doctrine\Common\Persistence\Mapping\MappingException: "Class 'Acme\ApiBundle\Entity\User' does not exist" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96 {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 18:29:25] request.CRITICAL: Exception thrown when handling an exception (Doctrine\Common\Persistence\Mapping\MappingException: Class 'Acme\ApiBundle\Entity\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php line 96) {"exception":"[object] (Doctrine\\Common\\Persistence\\Mapping\\MappingException(code: 0): Class 'Acme\\ApiBundle\\Entity\\User' does not exist at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/vendor/doctrine/common/lib/Doctrine/Common/Persistence/Mapping/MappingException.php:96)"} []
[2016-10-14 22:40:48] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/user/activate" (from "http://localhost/ivq-rest/web/api/doc")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/user/activate\" (from \"http://localhost/ivq-rest/web/api/doc\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:140)"} []
[2016-10-14 22:41:30] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/user/activate" (from "http://localhost/ivq-rest/web/api/doc")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/user/activate\" (from \"http://localhost/ivq-rest/web/api/doc\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:140)"} []
[2016-10-14 22:42:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/user/activate" (from "http://localhost/ivq-rest/web/api/doc")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/user/activate\" (from \"http://localhost/ivq-rest/web/api/doc\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:140)"} []
[2016-10-14 22:42:11] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/user/activate" (from "http://localhost/ivq-rest/web/api/doc")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/user/activate\" (from \"http://localhost/ivq-rest/web/api/doc\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:140)"} []
[2016-10-14 22:42:47] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /app/user/activate" (from "http://localhost/ivq-rest/web/api/doc")" at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php line 2605 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /app/user/activate\" (from \"http://localhost/ivq-rest/web/api/doc\") at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/classes.php:2605, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/htdocs/ivq-rest/app/cache/prod/appProdUrlMatcher.php:140)"} []
